csgo_bot.py

```diff
@@ -23,7 +23,7 @@
 # CSGO -1001107551770
 GROUP_ID = -1001107551770
 
-ADMINS = [6879883, 15418061, 150147251, 258786599, 1346477, 264856075, 39284761, 2622857]
+ADMINS = [u.telegram_id for u in db.get_admins()]
```

Comment:
how are you going to set admin rights(1).

Reply:
I've add 3 new commands /get_admins /set_admin and /revoke_admin to add new admin, in order to do that you should start the bot with the param -a telegram_id.

Indeed I need to parse the message and add the passed telegram_id to admins not the super_admin...